crates/egui_extras/src/table.rs

```diff
@@ -409,6 +411,15 @@ impl<'a> TableBuilder<'a> {
         self
     }
 
+    /// Should the scroll area animate `scroll_to_*` functions?
+    ///
+    /// Default: `true`.
+    #[inline]
+    pub fn animated(mut self, animated: bool) -> Self {
+        self.scroll_options.animated = animated;
```

Comment:
Let's call this `animate_scrolling` to be more explicit

```suggestion
    pub fn animate_scrolling(mut self, animated: bool) -> Self {
        self.scroll_options.animated = animated;
```